### ISSUE TITLE
Please mention the validation set in README.md

### ISSUE THREAD

**Issue by Prabhat-IIT** [manideep2510/eye-in-the-sky#2]
Hi, I think everyone who presented in this competition just played it dirty by not mentioning the validation set. I'd like to request that you mention the images taken in the validation set so that reader will not be misled by the results if one tries to reproduce them. Btw nice code  we'll also make our code public soon :)

**Comment by manideep2510** [comment]
Hi Prabhat,
I have included the information about the training and validation dataset in the README.
**We have used the first 13 images as training set and the last image in the dataset as the validation set**. 
Please refer to README for further information about the split.
Thanks.
